### ISSUE TITLE
KeepEmpty not working

### ISSUE THREAD

**Issue by chovy** [suisho/gulp-unzip#12]
I get an error and it unzips to an empty file rather than a directory.

```
[11:28:29] 'unzip-repo' errored after 9.12 ms
[11:28:29] Error: EEXIST, mkdir '/Users/username/projects/example/plugin.video.example-1.0'
```

The zip file contains a few empty directories which I want to keep intact.

**Comment by suisho** [comment]
Can you see your gulpfile?

**Comment by inuscript** [comment]
Fixed #14
